docs/spec/file-structures.md

```diff
@@ -45,40 +44,40 @@ Anchor Files contain [Create](#create), [Recover](#recover), and [Deactivate](#d
 ```
 :::
 
-A valid [Anchor File](#anchor-file) is a JSON document that ****MUST NOT**** exceed the [`MAX_ANCHOR_FILE_SIZE`](#max-anchor-file-size), and composed as follows:
+A valid [Core Index File](#core-index-file) is a JSON document that ****MUST NOT**** exceed the [`MAX_ANCHOR_FILE_SIZE`](#max-core-index-file-size), and composed as follows:
```

Comment:
MAX_ANCHOR_FILE_SIZE should be changed as well

Reply:
Addressed in new commit